src/main/kotlin/com/sainsburys/k2zpl/command/FieldBlock.kt

```diff
@@ -82,6 +82,6 @@ fun ZplBuilder.fieldBlock(
         alignment = alignment,
         hangingIndent = hangingIndent
     )
-    fieldData(data)
+    fieldData(data, replaceNewlines = true)
```

Comment:
I almost wonder if we can get away with making all `fieldData` usages replace newlines, and in turn eliminate this branch in the spirit of YAGNI.

WYDT?

Reply:
Good point, I don't see a time when we'd want to allow newlines specifically on this command type.

Reply:
Updated this one as well 👍